x-pack/test/detection_engine_api_integration/security_and_spaces/tests/utils.ts

```diff
@@ -118,6 +120,7 @@ export const getSimpleRuleOutput = (ruleId = 'rule-1'): Partial<OutputRuleAlertR
   false_positives: [],
   from: 'now-6m',
   immutable: false,
+  index: ['auditbeat-*'],
```

Comment:
Per comments above, I would not break API compatibility with these. These should be reverted and instead we still default these unless they are of the new type which is machine learning.

Reply:
Leaving this change as it's exercising the ability of the client to set an input index, independent of the backend's defaulting functionality.